starter_code/public/stylesheets/styles.css

```diff
@@ -5,4 +5,114 @@ dark-blue: #0A2C42;
 grey: rgba(145, 145, 145, 0.481);
 font-family: Arial, Helvetica, sans-serif
 */
-  
\ No newline at end of file
+/*RESET*/
+* {
+  margin: 0;
+  padding: 0;
+}
+body {
+  height: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+  background: rgba(145, 145, 145, 0.481);
+}
+/*NAVIGATION & FOOTER*/
+nav {
+  background: #2081C3;
+  position: relative;
+  overflow: hidden;
+}
+nav ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  list-style: none;
+  padding: 0 2rem;
+}
+nav ul li {
+  padding: 1rem;
+}
+.nav-li {
+  text-decoration: none;
+  color: white;
+}
+footer {
+  position: fixed;
+  bottom:0;
+  width:100%;
+  height:2rem; 
+  font-size: .5rem;
+  color: white;
+  background:#0A2C42;
+}
+footer h1 {
+  padding: 0 1rem;
+}
+/*LANDING PAGE*/
+.index-div {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem;
+}
+.index-div button {
+  padding: .5rem 1rem;
+  margin: 1rem;
+  border-radius: 5px;
+  background: #2081C3;
+}
+.index-button {
+  text-decoration: none;
+  color: white;
+}
+/*BEER PAGE*/
+.beer-page {
+  display: grid;
+  grid-template-columns: 33% 33% 33%;
+  grid-template-rows:min-content;
+  margin: 5%;
+}
+.beer-div {
+  margin: 5%;
+}
+.beer-img {
+  height: 20em;
+  margin: 2.5%;
+}
+.image-div {
+  text-align: center;
+}
+/*RANDOM BEER PAGE*/
+.random-beer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  margin: 10% 20%;
+}
+.random-beer img {
+  height: 25rem;
+  width: auto;
+}
+.random-image {
+  margin: 0 10%;
+}
+.random-beer p {
+  margin: 1%;
+}
+.random-beer h1 {
+  margin: 1%;
```

Comment:
Maybe there is a way to fix this margin thing without having to create so many rules?